adaptive

bragatudorn opened the pull request.

bragatudorn committed:
adaptive

Nejdancs commented on src/partials/contacts.html:
```diff
@@ -1,78 +1,83 @@
-          <p class="contacts__open-time__hours border-after">08:00 AM - 16:00 PM</p>
-        </div>
-        <address>
+        <ul class="list contacts__open-time">
+          <li class="contacts__open-time__days">Monday - Friday</li>
+          <li class="contacts__open-time__hours contacts__open-time--margin">06:00 AM - 10:00 PM</li>
+          <li class="contacts__open-time__days">Saturday - Sunday</li>
+          <li class="contacts__open-time__hours">08:00 AM - 16:00 PM</li>
+        </ul>
```
ты неправильно понял немного) я имел ввиду 2 лишки и в них засунуть п которые были.  и класс маржин не нужен будет. просто задай маржин лишке через нот ласт чаилд

    <ul class="list contacts__time-list">
          <li class="contacts__time-item">
             <p class="contacts__open-time-days">Monday - Friday</p>
             <p class="contacts__open-time-hours">06:00 AM - 10:00 PM</p>
          </li>
          <li class="contacts__time-item">
            <p class="contacts__open-time-days">Saturday - Sunday</p>
           <p class="contacts__open-time-hours">08:00 AM - 16:00 PM</p>
          </li>
     </ul>

bragatudorn replied:
теперь понял :D спасибо!

Nejdancs commented on src/sass/layout/_contacts.scss:
```diff
@@ -1,108 +1,172 @@
-.contact-section {    
-    background:#FFF5F6;    
+.contacts {    
+    background:#FFF5F6;  
+    padding: 115px 0px;
 }
```
Пересмотри падинги на всех экранах. делай впритык к соседним секциям Пример: padding-top 86px padding-bottom 113px для мобилки

bragatudorn replied:
у футера уже есть падинг 50 получается мне добавить просто остальное правильно?

Nejdancs commented on src/sass/layout/_contacts.scss:
```diff
@@ -1,108 +1,172 @@
+    flex-direction: column;
+    padding-bottom: 54px; 
+    @include tablet{flex-direction:initial; }
+
```
padding-bottom: 81px; для таблетки

Nejdancs commented on src/sass/layout/_contacts.scss:
```diff
@@ -1,108 +1,172 @@
 }
-.contacts {
-    display: flex;    
+.contacts__list {
+    @include flex(flex,center,center);
+    flex-direction: column;
+    padding-bottom: 54px; 
+    @include tablet{flex-direction:initial; }
+
+    @include desktop{        
+    padding-bottom: 86px; 
+    }
+         
 }
```
Мы договаривались отритцательным марженом делать сетки.

.contacts__list {
  margin-top: -20px;

  @include tablet {
    @include flex;
    flex-wrap: wrap;
    margin: -10px;
  }

  @include desktop {
    margin: -15px;
  }
}

bragatudorn replied:
ок спасибо!

Nejdancs commented on src/sass/layout/_contacts.scss:
```diff
@@ -1,108 +1,172 @@
+    padding: 43px 20px;
+    width: 220px;
+    height: 427px;   
     background: #FFFFFF;
     box-shadow: 0px 8px 30px rgba(212, 20, 67, 0.1);
-    border-radius: 24px; 
-    padding: 51px 43px;  
-    &:not(last-child){
-     margin-left: 31px;   
-    }   
+    border-radius: 24px;     
+    margin-bottom: 20px;
+    @include tablet{
+        width: 220px;
+        height: 544px;
+        padding: 50px 31px;
+        &:not(:last-child) {
+            margin-right: 20px;
+            }  
+    }  
+    @include desktop{ 
+        width: 330px;
+        height: 572px; 
+        padding: 51px 43px;       
+        &:not(:last-child) {
+        margin-right: 30px;
+        }    
+    }
+    
     
 }
```
.contacts__item {
  padding: 43px 20px;
  width: 220px;
  margin-top: 20px;
  margin-left: auto;
  margin-right: auto;

  background: #ffffff;
  box-shadow: 0px 8px 30px rgba(212, 20, 67, 0.1);
  border-radius: 24px;

  @include tablet {
    width: calc(100% / 3 - 20px);
    margin: 10px;
    padding: 50px 31px;
  }
  @include desktop {
    width: calc(100% / 3 - 30px);
    margin: 15px;
    padding: 51px 43px;
  }
}

bragatudorn replied:
спасибо!

Nejdancs commented on src/sass/layout/_contacts.scss:
```diff
@@ -1,108 +1,172 @@
+         
 }
+
+.contacts__buttons{
+    @include flex(flex,center,center);
+    flex-direction: column;
+    @include tablet{
+        flex-direction:initial; 
+    }
+          
+}
```
Нужно для мобилки  как можно меньше флексов делать

.contacts__buttons {
  margin-top: 54px;
  text-align: center;

  @include tablet {
    @include flex(flex, center, center);
    margin-top: 81px;
  }

  @include desktop {
    margin-top: 86px;
  }
}

Nejdancs commented on src/sass/layout/_contacts.scss:
```diff
@@ -1,108 +1,172 @@
+    padding: 43px 20px;
+    width: 220px;
+    height: 427px;   
     background: #FFFFFF;
     box-shadow: 0px 8px 30px rgba(212, 20, 67, 0.1);
-    border-radius: 24px; 
-    padding: 51px 43px;  
-    &:not(last-child){
-     margin-left: 31px;   
-    }   
+    border-radius: 24px;     
+    margin-bottom: 20px;
+    @include tablet{
+        width: 220px;
+        height: 544px;
+        padding: 50px 31px;
+        &:not(:last-child) {
+            margin-right: 20px;
+            }  
+    }  
+    @include desktop{ 
+        width: 330px;
+        height: 572px; 
+        padding: 51px 43px;       
+        &:not(:last-child) {
+        margin-right: 30px;
+        }    
+    }
```
.contacts__item {
  padding: 43px 20px;
  width: 220px;
  margin-top: 20px;
  margin-left: auto;
  margin-right: auto;

  background: #ffffff;
  box-shadow: 0px 8px 30px rgba(212, 20, 67, 0.1);
  border-radius: 24px;

  @include tablet {
    width: calc(100% / 3 - 20px);
    margin: 10px;
    padding: 50px 31px;
  }
  @include desktop {
    width: calc(100% / 3 - 30px);
    margin: 15px;
    padding: 51px 43px;
  }
}

Nejdancs commented on src/sass/layout/_contacts.scss:
```diff
@@ -1,108 +1,172 @@
+    } 
 }
 
 .contacts__open-time__hours{
```
Подумай над классами немного по бэму .contacts__open-time__hours так по бэм нельзя делать как вариан так можно contacts__open-time-hours

bragatudorn replied:
понял спасибо!

bragatudorn committed:
feature-contacts

bragatudorn committed:
contacts

ул-список итемов должен вылезать на верхний сектор? не уверен будет ли правильно сделать так
